modules/webadmin.cpp

```diff
@@ -1077,7 +1077,10 @@ class CWebAdminMod : public CModule {
             WebSock.PrintErrorPage(t_s("Network name is a required argument"));
             return true;
         }
-        if (!pNetwork || pNetwork->GetName() != sName) {
+        if (pNetwork && pNetwork->GetName().Equals(sName)) {
+            // Set the network name to pickup changes to capitalization.
+            pNetwork->SetName(sName);
```

Comment:
This skips the rest of logic of renaming network, like moving files to the new place or calling `OnAddNetwork`/`OnDeleteNetwork` of modules

Reply:
1. Copy [this](https://github.com/znc/znc/blob/6cd80535c064e91155468e79f50fc1c3f3c0f3e5/modules/webadmin.cpp#L1089) up here?
2. Should this mimic `AddNetwork`/`DeleteNetwork` behavior and make direct `USERMODULECALL`s?

Reply:
Could it reuse the logic instead of copying or mimicking it?

Reply:
Maybe with an optional `bool renaming = false` parameter on `AddNetwork`/`DeleteNetwork`.

If only capitalization changed, should that count as a new network?
Are module paths case-sensitive?